### PR TITLE
chore: Import local nns frontend canister IDs

### PR DESCRIPTION
**Pull request by bitdivine**
# Description
`dfx nns import` sets the local canister IDs for many but not all the NNS canisters.  This PR adds the `nns-dapp` and `internet_identity` to the canisters covered by this command.

# How Has This Been Tested?
There was already a test, commented out.  It just had to be uncommented.

# Checklist:

- [x] The title of this PR complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] I have edited the CHANGELOG accordingly.
- [x] I have made corresponding changes to the documentation. (unaffected)

### CODE REVIEW

File: src/dfx/src/commands/nns/import.rs [hunk 1]
```diff
@@ -61,14 +61,20 @@ pub async fn exec(env: &dyn Environment, opts: ImportOpts) -> DfxResult {
 /// The "local" entries at the remote URL are often misssing or do not match our NNS installation.
 /// Always set the local values per our local NNS deployment.  We have all the information locally.
 fn set_local_nns_canister_ids(logger: &Logger, config: &mut Config) -> DfxResult {
```
Comment (bitdivine) [inline src/dfx/src/commands/nns/import.rs:63]: Note: `NNS_CORE` and `NNS_FRONTEND` have different types so we cannot simply do `NNS_CORE.iter().chain(NNS_FRONTEND.iter())`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2545,4 +2548,4 @@ FLAGS:
 
 ## Agents
 
-## Canister Development Kit (CDK)
```
Comment (bitdivine) [inline CHANGELOG.md:2548]: I see no difference - newline maybe?  No change is intended here.

File: src/dfx/src/commands/nns/import.rs
Comment (bitdivine) [inline src/dfx/src/commands/nns/import.rs:1]: This file implements `dfx nns import`.  But previously it covered only some of the canisters deployed by `dfx nns install`.  Specifically, it covered the backend canisters such as governance and ledger but not the frontend canisters such as internet identity and the nns-dapp.  So now, we add the frontend canisters.

File: CHANGELOG.md [hunk 3]
```diff
@@ -29,6 +29,9 @@ The options are "cycles", "size", "O4", "O3", "O2", "O1", "O0", "Oz", and "Os".
 - Updates the starter project for env vars to use the new `dfx build` & `dfx deploy` environment variables
 - Changes the format of the canister id env vars to be `CANISTER_ID_<canister_name_uppercase>`, for the frontend declaraction file to be consistent with the dfx environment variables. `CANISTER_ID` as both a prefix and suffix are supported for backwards compatibility.
 
+### chore: Add the `nns-dapp` and `internet_identity` to the local canister IDs set by `dfx nns import`
+`dfx nns install` installs a set of canisters in a local repolica.  `dfx nns import` complements this by setting the canister IDs so that they can be queried by the user.  But `dfx nns import` is incomplete.  Now it will also provide the IDs of the `nns-dapp` and `internet_identity` canisters.
+
```
Comment (sesi200) [inline CHANGELOG.md:34]: Merging `master` put this in the `0.14.0` section - please move it back to unreleased

Comment (bitdivine) [reply CHANGELOG.md]: Done.  Does that mean that 0.14.0 is about to be released? (With bated breath.)  We should give the releases codenames.  More fun.

**Review by sesi200 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (sesi200) [reply CHANGELOG.md]: Yes, it's about to be promoted

**Pull request merged**